packages/generate-bot/test/generate-bot.js

```diff
@@ -146,4 +134,16 @@ describe("user input", () => {
     const regexLinux = new RegExp("packages/pass");
     expect(regexLinux.test(fileLocation)).to.be.true;
   });
+
+  it("checks that hyphens are turned into underscores", () => {
+    const hyphenTest = {
+      programName: "pass-but-change",
+      description: "pass",
```

Comment:
it would probably also be good to make sure that we allow someone to provide the name `foo_bar` for a bit, i.e., that we allow underscores.

Reply:
Thanks for the review @bcoe! Implemented the logic we discussed for module.exports['program-name'] to allow for hyphens. I'll also submit a PR to Colin's script to encourage the use of hyphens as well!